runtime/j9vm/java11vmi.c

```diff
@@ -280,11 +275,7 @@ getModuleObjectClassLoader(J9VMThread * currentThread, j9object_t moduleObject)
 static j9object_t
 getModuleObjectName(J9VMThread * currentThread, j9object_t moduleObject)
 {
-	if(J2SE_SHAPE(currentThread->javaVM) < J2SE_SHAPE_B165) {
-		return J9VMJAVALANGREFLECTMODULE_NAME(currentThread, moduleObject);
-	} else {
-		return J9VMJAVALANGMODULE_NAME(currentThread, moduleObject);
-	}
+	return J9VMJAVALANGMODULE_NAME(currentThread, moduleObject);
```

Comment:
The method `getModuleObjectName` can be folded completely.

Reply:
Done.